src/lib/fields/LanguagesField.js

```diff
@@ -0,0 +1,28 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import _get from 'lodash/get';
+import _set from 'lodash/set';
+import _cloneDeep from 'lodash/cloneDeep';
+
+import { Field } from './Field';
+
+export class LanguagesField extends Field {
+  deserialize(record) {
+    let langu = _get(record, this.fieldpath, this.deserializedDefault);
+    identifiers = identifiers.map((identifier) => ({}));
+    return record;
+  }
```

Comment:
I think something is not quite right here :grin: ...

Reply:
Oups, this is a leftover! You can delete it!

Reply:
Got it. I will take over this PR :+1: